src/components/decision/classification/classification.ts

```diff
@@ -1,28 +1,185 @@
+      // cannot derive a true shortcut
+      return;
+    }
+
+    return this.falseShortcut === this.falseShortcut.toUpperCase()
+      ? this.falseShortcut.toLowerCase()
+      : this.falseShortcut.toUpperCase();
+  }
+
+  protected get derivedFalseShortcut(): string | undefined {
+    if (this.falseShortcut) {
+      return this.falseShortcut;
+    } else if (!this.trueShortcut) {
+      return;
+    }
+
+    return this.trueShortcut === this.trueShortcut.toUpperCase()
+      ? this.trueShortcut.toLowerCase()
+      : this.trueShortcut.toUpperCase();
+  }
+
+  public override shortcutKeys(): KeyboardShortcut[] {
+    const shortcuts: KeyboardShortcut[] = [];
+
+    if (this.derivedTrueShortcut) {
+      shortcuts.push({
+        key: this.derivedTrueShortcut,
+        description: `Add a 'true' ${this.tag.text} classification`,
+      });
+    }
+
+    if (this.derivedFalseShortcut) {
+      shortcuts.push({
+        key: this.derivedFalseShortcut,
+        description: `Add a 'false' ${this.tag.text} classification`,
+      });
+    }
+
+    return shortcuts;
+  }
+
+  protected override handleShortcutKey(event: KeyboardEvent): void {
+    if (this.isTrueShortcutKey(event)) {
+      this.trueDecisionButton.click();
+      this.falseDecisionButton.focus();
+      return;
+    }
+
+    if (this.isFalseShortcutKey(event)) {
+      this.falseDecisionButton.click();
+      this.falseDecisionButton.focus();
+    }
+  }
+
+  protected override isShortcutKey(event: KeyboardEvent): Readonly<boolean> {
+    return this.isTrueShortcutKey(event) || this.isFalseShortcutKey(event);
+  }
+
+  private isTrueShortcutKey(event: KeyboardEvent): boolean {
+    if (!this.derivedTrueShortcut) {
+      return false;
+    }
+
+    return event.key === this.derivedTrueShortcut;
+  }
+
+  private isFalseShortcutKey(event: KeyboardEvent): boolean {
+    if (!this.derivedFalseShortcut) {
+      return false;
+    }
+
+    return event.key === this.derivedFalseShortcut;
+  }
+
+  private decisionButtonTemplate(decision: DecisionOptions, decisionId: number): TemplateResult {
+    const buttonClasses = classMap({
+      disabled: !!this.disabled,
+      "oe-btn-primary": decision === DecisionOptions.TRUE || decision === DecisionOptions.FALSE,
+      "oe-btn-secondary": decision === DecisionOptions.UNSURE || decision === DecisionOptions.SKIP,
+    });
+
+    const shortcut = decision === DecisionOptions.TRUE ? this.derivedTrueShortcut : this.derivedFalseShortcut;
+    const decisionModel = new Classification(decision, decisionId, this.tag);
+    const color = this.injector.colorService(decisionModel);
+
+    return html`
+      <button
+        id="${decision}-decision-button"
+        class="decision-button ${buttonClasses}"
+        part="${decision}-decision-button"
+        title="Shortcut: ${shortcut}"
+        style="--ripple-color: var(${color})"
+        aria-disabled="${this.disabled}"
```

Comment:
are we missing an ordinary disabled button?

Reply:
I don't quite understand what this is asking for

We do support disabling any element (including any buttons) through the `disabled` class in our themeing.css file